lib/services/PostServices.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutterapp/src/common/global.dart';
+import "package:gql_exec/gql_exec.dart";
+import "package:gql/language.dart";
+import 'package:gql_http_link/gql_http_link.dart';
+
+class PostService {
```

Comment:
Thanks for using GraphQL!
This class seems to manage the requests to the GraphQL endpoint.